tensorflow_addons/metrics/hamming_test.py

```diff
@@ -39,6 +40,7 @@ def update_obj_states(self, obj, actuals, preds):
     def check_results(self, obj, value):
         self.assertAllClose(value, self.evaluate(obj.result()), atol=1e-5)
 
+    @unittest.skip("Failing. See https://github.com/tensorflow/addons/issues/1200")
```

Comment:
@gabrieldemarmiesse  Just wondering what benefits we see for these PR's to use unittest.skip vs. the self method:
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/image/resampler_ops_test.py#L100

Don't really have a strong preference for one over the other, but it does prevent an additional import

Reply:
I don't have a strong preference either. I think they're equivalent. Do you want me to change them to `self.skip`?

Reply:
If you don't mind it'll be consistent with the other skips we've implemented

Reply:
Sure thing